pages/ksphere/kommander/latest/tutorials/use-kcl-from-kubectl/index.md

```diff
@@ -0,0 +1,339 @@
+kcl-tfcb-9c88c7bc9-mxfjp                                   1/1     Running     0          75m
+kcl-utility-apiserver-7ffcc5b4bf-jzd69                     1/1     Running     0          75m
+kcl-webhook-87f7b8d45-vqcm5                                1/1     Running     0          75m
+kubefed-admission-webhook-7575b6b474-hvctv                 1/1     Running     0          75m
+kubefed-controller-manager-76bcdf6dc4-lp9xr                1/1     Running     0          75m
+kubefed-controller-manager-76bcdf6dc4-zhm54                1/1     Running     0          75m
+```
+
+## Viewing KCL Logs
+
+```
+kubectl -n kommander logs -l control-plane=kcl-cm -c controller-manager
+```
+
+### Viewing KCL Webhook Logs
+
+```
+kubectl -n kommander logs -l control-plane=kcl-webhook
+```
+
+## Working with the Workspaces
+
+Learn more about [Workspaces][kommander_workspaces] in Kommander.
+
+### Show the list of the Workspaces
+
+```
+$ kubectl get workspaces
+
+NAME                     DISPLAY NAME             WORKSPACE NAMESPACE            AGE
+default-workspace        Default Workspace        default-workspace-4clss        77m
+```
+
+### Creating a new Workspace
+
+```
+cat - << EOF | kubectl apply -f -
+apiVersion: workspaces.kommander.mesosphere.io/v1alpha1
+kind: Workspace
+metadata:
+  name: workspacetest
+EOF
+```
+
+List workspaces again to get the Workspace namespace. We are going to need it further.
+
+```
+$ kubectl get workspaces
+
+NAME                     DISPLAY NAME             WORKSPACE NAMESPACE            AGE
+default-workspace        Default Workspace        default-workspace-4clss        95m
+workspacetest                                     workspacetest-r69q2            77s
+```
+
+## Creating a cluster in the Workspace
+
+### Creating an AWS Cloud Provider
+
+In this tutorial we are going to be using AWS Provider. Althouhg you can use any other supported [infrastructure provider][infrastructure_provider].
+
+<p class="message--note"><strong>NOTE: </strong>Assuming you logged in with aws-cli
+</p>
+
+```
+BASE64_ARGS="-w 0"
+if [ "$(uname)" == "Darwin" ]; then
+  BASE64_ARGS=
+fi
+
+CREDENTIALS=$( base64 ${BASE64_ARGS} ~/.aws/credentials )
+PROFILE=$(echo -n "${AWS_PROFILE}" | base64 ${BASE64_ARGS} )
+
+cat - << EOF | kubectl apply -f -
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: aws-credentials
+  namespace: workspacetest-r69q2
+data:
+  credentials: ${CREDENTIALS}
+  profile: ${PROFILE}
+type: kommander.mesosphere.io/aws-credentials
+---
+apiVersion: kommander.mesosphere.io/v1beta1
+kind: CloudProviderAccount
+metadata:
+  name: aws-credentials
+  namespace: workspacetest-r69q2
+  annotations:
+    kommander.mesosphere.io/display-name: My AWS Credentials
+spec:
+  provider: aws
+  credentialsRef:
+    name: aws-credentials
+EOF
+```
+
+### Creating a cluster
+
+To create a cluster via KCL means we ecreate a KonvoyCluster CRD in the Workspace namespace.
```

Comment:
```suggestion
To create a cluster using KCL means you ecreate a KonvoyCluster CRD in the Workspace namespace.
```

Reply:
Typo in `ecreate` and wdyt of pointing out we need the CloudProviderAccount?

```suggestion
To create a cluster using KCL means you create a KonvoyCluster CRD in the Workspace namespace. You need to reference the CloudProviderAccount with the AWS credentials you created in the previous step.
```